src/installer.ts

```diff
@@ -7,6 +7,7 @@ import * as tc from '@actions/tool-cache';
 import * as path from 'path';
 import * as semver from 'semver';
 import fs = require('fs');
+import * as installer from './installer';
```

Comment:
I'm a little confused this line: why we need to import `installer` to the `installer` file?

Reply:
left by mistake, removed